jsapp/js/app.jsx

```diff
@@ -139,7 +144,14 @@ class App extends React.Component {
               </bem.PageWrapper__content>
             </bem.PageWrapper>
           </RootContextProvider>
-        </>
+
+          {/* React Query Devtools. Development mode only.
+              Lower button opacity to be less distracting
+              if it shows up in a screenshot. */}
+          <style>{'.tsqd-open-btn-container { opacity: 0.1 !important; };'}</style>
+          <ReactQueryDevtools />
```

Comment:
As per our previous discussion, I think we agreed to comment this out and then merge the PR :)

Reply:
:heavy_check_mark:  [`da7576f` (#5001)](https://github.com/kobotoolbox/kpi/pull/5001/commits/da7576f07a60a27e0581c4fa9b51e79bbb9304ad)